### ISSUE TITLE
[New package version]: Mirillis.Action version 4.24.0

### ISSUE THREAD

**Issue by OfficialEsco** [microsoft/winget-pkgs#38195]
### Package Requested

- [x] I would like someone else to build the manifest.
- [x] I would like help so I can submit the manifest.
- [x] I have performed a search and couldn't find this package.
- [x] I have checked that the installer for this package is not a standalone or portable executable or a .zip file
- [X] I think there is a new version available and I have provided the URL.

### Please provide the following information

Publisher: Mirillis
Package Name: Action
Package Version: 4.24.0
Installer URL: https://downloads.mirillis.com/files/action_4_24_0_setup.exe

Current Package Identifier (for new versions): Mirillis.Action

No idea how to get past this
![image](https://user-images.githubusercontent.com/15158490/146656317-8766d046-d658-4875-b34b-edbf02f6fcf2.png)
🤷‍♂️
```
PS C:\Users\User\Downloads> .\action_4_24_0_setup.exe /S
PS C:\Users\User\Downloads> .\action_4_24_0_setup.exe /Silent
PS C:\Users\User\Downloads> .\action_4_24_0_setup.exe --silent
PS C:\Users\User\Downloads> .\action_4_24_0_setup.exe /S /v/q
PS C:\Users\User\Downloads> .\action_4_24_0_setup.exe /EXENOUI
PS C:\Users\User\Downloads> .\action_4_24_0_setup.exe /quiet
PS C:\Users\User\Downloads> .\action_4_24_0_setup.exe /q
PS C:\Users\User\Downloads> .\action_4_24_0_setup.exe --quiet
```
It does appear to be Nullsoft tho, however wingetcreate does not recognize it as that
![image](https://user-images.githubusercontent.com/15158490/146656313-301a0d56-0f39-4e5d-b726-c838dfa03d18.png)

**Comment by SpecterShell** [comment]
This installer is built by NSIS, though it is wrapped by another executable which cannot pass the parameter to the installer.

**Comment by kamack38** [comment]
Silent prams have stopped working since `4.24.0`

**Comment by kamack38** [comment]
It was a bug. They have changed the installer to allow administrator and non administrator installation and this removed passing the command line arguments. This is  now fixed in the newest version (`4.24.3`

**Comment by OfficialEsco** [comment]
> It was a bug. They have changed the installer to allow administrator and non administrator installation and this removed passing the command line arguments. This is now fixed in the newest version (`4.24.3`)

Thank you for the update =)
